toolkits/global/packages/global-lists/scss/10-settings/_springernature.scss

```diff
@@ -1,3 +1,12 @@
+/**
+ * List
+ * Springer branding for component
```

Comment:
should say "Springer Nature branding" here

Reply:
God damn it bloody copy paste, will fix it